docs/docs/dagster-plus/deployment/deployment-types/hybrid/azure/key-vault.md

```diff
@@ -0,0 +1,276 @@
+---
+title: "Retrieve secrets and credentials from Azure Key Vault in AKS"
+sidebar_position: 400
+---
+
+:::note
+This guide is applicable to Dagster+ on AKS only and requires version 1.10.2 or later of the Helm chart.
+:::
+
+In this guide, we'll walk through how to retrieve secrets and credentials from Azure Key Vault in an Azure Kubernetes Service (AKS) cluster. 
+This guide assumes you completed the first step of [Deploying Dagster+ hybrid on Azure](.). 
+
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+
+- The Azure CLI installed on your machine. You can download it [here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+- `kubectl` installed on your machine. You can download it [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+- `helm` installed on your machine. You can download it [here](https://helm.sh/docs/intro/install/).
+- A functional AKS cluster and Dagster+ agent running on it. If you have not yet set up an AKS agent, you can look up [Deploy an Azure Kubernetes Service (AKS) agent guide](aks-agent).
+
+
+## Step 1: Enable the Azure Key Vault provider for the AKS cluster
+
+To deploy the AKS Secrets store CSI drivers and providers, run the following command:
```

Comment:
nit; maybe it's just me but I think a bit more preamble here might be nice before we dive into service names. Something like; "we need to make sure that key vault is available as an addon to the aks cluster.